completion/catkin_tools-completion.bash

```diff
@@ -54,7 +56,7 @@ _catkin()
   if [[ ${COMP_CWORD} -eq 1 || ${catkin_opts} == *${prev}* ]] ; then
     COMPREPLY=($(compgen -W "${catkin_verbs}" -- ${cur}))
   else
-    if [[ "${COMP_WORDS[@]}" == *" build"* ]] ; then
+    if [[ "${COMP_WORDS[@]}" =~ ".* build|bt.*" ]] ; then
```

Comment:
This is very fragile. I really don't like the change to include `bt`. When it was `build` at least you knew that was the verb. With this change, this list would need to be extended anytime a new "build based" verb alias was added, otherwise option completion would not work.

The "right" way to do it would be to expand any verb aliases into a local variable (note not replacing it on the command line for the user) and checking what the "real verb" is based on that expansion. Then you could gate build option completion based on the real verb being build. This would work for any verb alias based on build. However, that sounds complicated, so I'd rather merge this without build options working with `bt`, i.e. remove the changes on this line. Then at least the verb aliases would auto complete, but we wouldn't have the inconsistent behavior that some aliases can complete but others cannot.